snyk/scan-python.sh

```diff
@@ -9,5 +9,9 @@ AGENT_NAME="$2"
 cp -r "$AGENT_PATH" /"$AGENT_NAME"
 cd /"$AGENT_NAME"
 if [ -f requirements.txt ]; then pip3 install -r requirements.txt >&2; fi
+pip3 uninstall -y inbm-lib
+grep -v inbm-lib <requirements.txt >tmp
```

Comment:
does the uninstall make it so that inbm-lib isn't scanned?  Or what is happening in these 4 lines?

Reply:
@nmgaston inbm-lib and its dependencies do get scanned, we just need to remove 'inbm-lib' from the list of installed packages because snyk confuses it with the malicious upstream pypi package 'inbm-lib'. We took some steps to make sure people downloading out source don't install inbm-lib from that source. Actually it shouldn't be in requirements.txt at all so I can remove that grep line.

Reply:
ok, done and pushed